### PR TITLE
Exist

### LINKED ISSUE #7: Feruz: Exists.js
# Repo
- [x]  developed on a branch named exists
# JS
- [x] finish the function exists.js

### PR DESCRIPTION
**Pull request by Feruzteame**
close #7 
Exist button

### CODE REVIEW

**Review by abelRoland — APPROVED** [review]
Perfect. Great job @Feruzteame . I didn't test it in my local, but I believe that works perfectly.

**Comment by Feruzteame** [comment]
Thank you abel !

**Review by yildiraykoyuncu — APPROVED** [review]
Nice work!

**Review by aleks2407 — APPROVED** [review]
thanks!

File: scripts/user-actions/exists.js [hunk 1]
```diff
@@ -3,18 +3,18 @@ console.log('--- loading: exists.js');
...
+    return entry === userInput
   }
-  const itExists = words._(_);
+  const itExists = words.find(equalsUserInput);
   if (itExists) {
-    alert(``);
+    alert(`"${userInput}" exists`);
   } else {
-    alert(``);
+    alert(`"${userInput}" does not exists`);
```
Comment (aleks2407) [inline scripts/user-actions/exists.js]: a little typo , should be `does not exist`

**Review by aleks2407 — CHANGES_REQUESTED** [review]

File: scripts/user-actions/exists.js (hunk 1 above)
Comment (Feruzteame) [reply scripts/user-actions/exists.js]: thank you @aleks2407 . i will fix it !

**Review by aleks2407 — APPROVED** [review]

**Pull request merged**